### PR TITLE
Add cypress test for cart event - part of USF 1654

### PR DESCRIPTION
**Pull request by cctina516**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://develop--aem-boilerplate-commerce--hlxsites.aem.live/
- After: https://cart-event-test--aem-boilerplate-commerce--hlxsites.aem.live/

### CODE REVIEW

File: cypress/src/tests/e2eTests/events/add-to-cart.spec.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { products } from "../../../fixtures";
+import { expectsEventWithContext } from "../../../assertions";
+/**
+ * https://github.com/adobe/commerce-events/blob/main/examples/events/initiate-checkout.md
```
Comment (sirugh) [inline cypress/src/tests/e2eTests/events/add-to-cart.spec.js:4]: Make sure you change this to the add-to-cart doc :)

Comment (cctina516) [reply cypress/src/tests/e2eTests/events/add-to-cart.spec.js]: done!

File: cypress/src/tests/e2eTests/events/add-to-cart.spec.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { products } from "../../../fixtures";
+/**
+ * https://github.com/adobe/commerce-events/blob/main/examples/events/add-to-cart.md
+ *
+ * Required Contexts: page, storefront, shoppingCart
```
[the remark this answers is not in the document]
Comment (cctina516) [reply cypress/src/tests/e2eTests/events/add-to-cart.spec.js]: I've made an update to include all the required contexts. Can you please take another look to see if I'm on the right track? Thanks.

**Review by sirugh — APPROVED** [review]
Yep, exactly :)

**Comment by anthoula** [comment]
Cypress failures for `events/recs.spec.js` exist already on the `develop` branch and are not due to these changes.

**Comment by anthoula** [comment]
retest

**Pull request merged**